packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.html

```diff
@@ -34,4 +35,78 @@
             </div>
         </div>
     </div>
+
+    <!-- if unconfigured without OVHcloud domain -->
+    <div data-ng-if="ctrl.dkimForNoOvhCloud">
+        <div
+            data-wizard
+            data-wizard-on-finish="leaveDkimConfigurator"
+            data-wizard-on-cancel="leaveDkimConfigurator"
+            data-wizard-hide-previous-button="true"
+            data-wizard-confirm-button-text=":: 'emailpro_tab_domain_diagnostic_dkim_confirm_no_ovhcloud' | translate"
+            data-wizard-next-button-text="getNextButtonDependingOnStep() | translate"
+            data-wizard-title="getTitleDependingOnStep() | translate"
+        >
+            <div data-wizard-step data-wizard-step-on-load="getDkimSelector">
+                <oui-spinner data-ng-if="ctrl.loading"></oui-spinner>
+
+                <div data-ng-if="!ctrl.loading">
+                    <p
+                        data-translate="emailpro_tab_domain_diagnostic_dkim_activation_no_ovhcloud"
+                    ></p>
+                </div>
+            </div>
+
+            <div
+                data-wizard-step
+                data-wizard-step-on-load="loadDataForDkim"
+                data-wizard-step-on-next="getDkimSelector"
+                data-wizard-step-valid="ctrl.isStepConfigureValid"
+            >
+                <div class="col-md-12" data-ng-if="ctrl.loading">
+                    <oui-spinner></oui-spinner>
+                </div>
+
+                <p
+                    data-translate="emailpro_tab_domain_diagnostic_dkim_id_generation_no_ovhcloud"
+                ></p>
+            </div>
+
+            <div
+                data-wizard-step
+                data-wizard-step-on-load="initSpfContext"
+                data-wizard-step-on-next="leaveDkimConfigurator"
+            >
+                <p
+                    data-translate="emailpro_tab_domain_diagnostic_dkim_configurate_dns_records_info"
+                ></p>
+
+                <dl class="dl-horizontal dl-md">
+                    <dt
+                        data-translate="emailpro_tab_domain_diagnostic_dkim_text_field"
+                    ></dt>
+                    <dd>
+                        <span data-ng-bind="ctrl.selector1NameInfos"></span>
+                    </dd>
+                    <dd>
+                        <span data-ng-bind="ctrl.selector1RecordInfos"></span>
+                    </dd>
+                    <dd>
+                        <span data-ng-bind="ctrl.selector2NameInfos"></span>
+                    </dd>
+                    <dd>
+                        <span data-ng-bind="ctrl.selector2RecordInfos"></span>
+                    </dd>
+                </dl>
```

Comment:
Refer https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-content-descriptions--horizontal
```suggestion
                <dl class="oui-description oui-description_horizontal">
                    <dt
                        data-translate="emailpro_tab_domain_diagnostic_dkim_text_field"
                    ></dt>
                    <dd>
                        <span data-ng-bind="ctrl.selector1NameInfos"></span>
                    </dd>
                    <dd>
                        <span data-ng-bind="ctrl.selector1RecordInfos"></span>
                    </dd>
                    <dd>
                        <span data-ng-bind="ctrl.selector2NameInfos"></span>
                    </dd>
                    <dd>
                        <span data-ng-bind="ctrl.selector2RecordInfos"></span>
                    </dd>
                </dl>
```